### ISSUE TITLE
Is there a way to get the answer to the question about the current page change?

### ISSUE THREAD

**Issue by fengzhihenxs** [surveyjs/survey-library#2335]
Some questions may have been answered on the page of my survey and stored on the server. If I change the questions on the page, including the answered and unanswered questions, how can I get the answers to the changed questions?

**Comment by andrewtelnov** [comment]
@fengzhihenxs I am not sure what is the goal. You can track two events: ``survey.onCurrentPageChanged`` and ``survey.onValueChanged``. You can put into your hash question that has been changed using ``onValueChanged`` event and clear it ``onCurrentPageChanged``.

Thank you,
Andrew

**Comment by fengzhihenxs** [comment]
@andrewtelnov Hello, I mean a comment to get the answer to the page change question. For example, I click a button after changing some questions to get the changed value. The onCurrentPageChanged event should not meet

**Comment by andrewtelnov** [comment]
@fengzhihenxs Sorry, what do you mean: " a comment to get the answer to the page change question"?

Thank you,
Andrew

**Comment by fengzhihenxs** [comment]
@andrewtelnov  No, if I have a button，this button to get the answer to the page change question，what to do?
I can only think of using the onValueChanged event to store the name and value of the changed question. Do you have a good way?
